src/main/java/io/github/moulberry/notenoughupdates/profileviewer/trophy/TrophyFishingPage.java

```diff
@@ -197,18 +198,30 @@ public static void renderPage(int mouseX, int mouseY) {
 		);
 
 		ArrayList<TrophyFish> arrayList = new ArrayList<>(trophyFishList.values());
-		arrayList.sort((c1, c2) -> {
-			if (c1.getTotal() > c2.getTotal()) return -1;
-			if (c1.getTotal() < c2.getTotal()) return 1;
-			return 0;
-		});
+		arrayList.sort((c1, c2) -> Integer.compare(c2.getTotal(), c1.getTotal()));
```

Comment:
Comparators.comparingInt lol

Reply:
i didn't know that existed, good to know tho